### ISSUE TITLE
🚧 🐞 Bug Report: Uncaught TypeError: Argument 1 passed to Elementor\Controls_Stack::sanitize_settings() must be of the type array, string given

### ISSUE THREAD

**Issue by HussienShaker** [elementor/elementor#20446]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Hello all,
I have a wordpress website and it was working 100% but yesterday its shows white screen 
when i try to edit pages with elementor it's show me ( The preview could not be loaded ... click here for preview debug )

And when i click the link its appear this lines 

Uncaught TypeError: Argument 1 passed to Elementor\Controls_Stack::sanitize_settings() must be of the type array, string given, called in /home1/greenheightsedu/public_html/Language/wp-content/plugins/elementor/includes/base/controls-stack.php on line 1107 and defined in /home1/greenheightsedu/public_html/Language/wp-content/plugins/elementor/includes/base/controls-stack.php:2412
Stack trace:
#0 /home1/greenheightsedu/public_html/Language/wp-content/plugins/elementor/includes/base/controls-stack.php(1107): Elementor\Controls_Stack->sanitize_settings('dreen HeightsLa...')
#1 /home1/greenheightsedu/public_html/Language/wp-content/plugins/elementor/includes/base/controls-stack.php(2161): Elementor\Controls_Stack->get_data('settings')
#2 /home1/greenheightsedu/public_html/Language/wp-content/plugins/elementor/core/base/base-object.php(142): Elementor\Controls_Stack->get_init_settings()
#3 /home1/greenheightsedu/public_html/Language/wp-content/plugins/elementor/core/base/base-object.php(40): Elementor\Core\Base\Ba

i try to delete all plugins one by one and still stuck on this error
Any Help Please ? 

### Steps to reproduce

delete all plugins one by one

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

<html><body>
<h3 class="wp-heading-inline">System Info</h3>
			<div>	<div class="elementor-system-info-section elementor-system-info-server">
		

Server Environment |   |  
-- | -- | --
Operating System: | Linux |  
Software: | Apache |  
MySQL version: | Percona Server (GPL), Release 23, Revision 500fcf5 v5.7.23-23 |  
PHP Version: | 7.4.33 |  
PHP Memory Limit: | 512M |  
PHP Max Input Vars: | 1000 |  
PHP Max Post Size: | 516M |  
GD Installed: | Yes |  
ZIP Installed: | Yes |  
Write Permissions: | All right |  
Elementor Library: | Connected |  

</div></div>
</body>
</html>System Info

			
			
				Server Environment
					
					
			

			
			
						
				Operating System:
					Linux
					
			

						
				Software:
					Apache
					
			

						
				MySQL version:
					Percona Server (GPL), Release 23, Revision 500fcf5 v5.7.23-23
					
			

						
				PHP Version:
					7.4.33
					
			

						
				PHP Memory Limit:
					512M
					
			

						
				PHP Max Input Vars:
					1000
					
			

						
				PHP Max Post Size:
					516M
					
			

						
				GD Installed:
					Yes
					
			

						
				ZIP Installed:
					Yes
					
			

						
				Write Permissions:
					All right
					
			

						
				Elementor Library:
					Connected
					
			

						
		

			
			
				WordPress Environment
					
					
			

			
			
						
				Version:
					6.1.1
					
			

						
				Site URL:
					https://greenheights.edu.eg/Language
					
			

						
				Home URL:
					https://greenheights.edu.eg/Language
					
			

						
				WP Multisite:
					No
					
			

						
				Max Upload Size:
					512 MB
					
			

						
				Memory limit:
					512M
					
			

						
				Max Memory limit:
					512M
					
			

						
				Permalink Structure:
					/%postname%/
					
			

						
				Language:
					en-GB
					
			

						
				Timezone:
					0
					
			

						
				Admin Email:
					greenheights77@gmail.com
					
			

						
				Debug Mode:
					Inactive
					
			

						
		

			
			
				Theme
					
					
			

			
			
						
				Name:
					Basic
					
			

						
				Version:
					1.3.12
					
			

						
				Author:
					WP Puzzle
					
			

						
				Child Theme:
					No
					If you want to modify the source code of your theme, we recommend using a [child theme](https://go.elementor.com/wordpress-child-themes/).
			

						
		

			
			
				User
					
					
			

			
			
						
				Role:
					admin
					
			

						
				WP Profile lang:
					en_GB
					
			

						
				User Agent:
					Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:106.0) Gecko/20100101 Firefox/106.0
					
			

						
		

			
			
				Active Plugins
					
					
			

			
			
							
					[Elementor](https://elementor.com/?utm_source=wp-plugins&utm_campaign=plugin-uri&utm_medium=wp-dash) - 3.8.0
						By [Elementor.com](https://elementor.com/?utm_source=wp-plugins&utm_campaign=author-uri&utm_medium=wp-dash)
						
				

								
					[Elementor Pro](https://go.elementor.com/wp-dash-wp-plugins-author-uri/) - 3.8.0
						By [Elementor.com](https://go.elementor.com/wp-dash-wp-plugins-author-uri/)
						
				

								
					[File Manager Advanced](https://wordpress.org/plugins/file-manager-advanced) - 5.0
						By [modalweb](https://advancedfilemanager.com/)
						
				

							
		

			
			
				Elementor Experiments
					
					
			

			
			
						
				
					
Optimised DOM Output: 	Active by default
Improved asset loading: 	Inactive by default
Improved CSS Loading: 	Inactive by default
Inline Font Icons: 	Inactive by default
Accessibility improvements: 	Active by default
Additional Custom Breakpoints: 	Inactive by default
Import Export Website Kit: 	Active by default
Hide native WordPress widgets from search results: 	Active by default
admin_menu_rearrangement: 	Inactive by default
Flexbox Container: 	Inactive by default
Default to New Theme Builder: 	Active by default
Landing pages: 	Active by default
Color Sampler: 	Active by default
Favorite Widgets: 	Active by default
Admin Top Bar: 	Active by default
Page Transitions: 	Active by default
Notes: 	Active by default
Loop: 	Inactive by default
Form Submissions: 	Active by default
Scroll Snap: 	Active by default

**Comment by nicholaszein** [comment]
Hey @HussienShaker, 

Thank you for submitting your issue! 🙏 

### ❌ While reviewing your report, we noticed it was missing some key pieces of information.

- **Prerequisites:** please search the repository before submitting an issue;
- **Steps to replicate:** You did not provide any way for us to try to reproduce your issue;
- **Isolating the Problem:** You need to make sure this only happens while using Elementor, Elementor Pro, and the Hello Elementor theme. We do not provide support for 3rd-party themes and plugins.
- **System info:** You did not provide any information about your [System Info](https://elementor.com/help/systems-info/) as outlined in our [guidelines](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml). Please provide the full **System Info** as available in your Elementor > System Info page. Scroll down to the end and download the file.

Without them, we simply can't help. 😕 

👉 Please follow the guidelines for opening a new [🐞 Bug Report](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml), so we can help you better.


### ⚠️ Note that GitHub issues are reserved for Elementor 🐞 Bug Reports only, and should not be used for support.

❔ If you have questions or need general support, please use one of the following channels:

- [Elementor Global Community](https://facebook.com/groups/Elementors/) on Facebook
- [Elementor Help Center](https://docs.elementor.com)
- [Support Forum](https://wordpress.org/support/plugin/elementor) on WordPress.org 
- [Elementor Developers Documentation](https://developers.elementor.com/)

💎 If you have purchased [Elementor Pro](https://elementor.com/pro), you are entitled to personal support, please see your purchase email or visit your [Elementor Account page](https://my.elementor.com/) for details.

Best regards

**Comment by nicholaszein** [comment]
Hello there, 

#### 🕙 It's been a while since we answered you, and we haven't received any feedback. 

## 🚧 For this reason, we are closing this issue because of no activity.

👉 If you wish to continue this discussion, please follow the guidelines for opening a new [🐞 Bug Report](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml).

👉 If you wish to continue this discussion, please follow the guidelines for opening a new [🚀 Feature Request](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=feature_request.yml).

Feel free to submit a new issue in accordance with these guidelines. 😉 

Kind regards
